From 36bcd36c00b4f556e9e79ace2782295a4f799d83 Mon Sep 17 00:00:00 2001
From: Kuaipz <2685697179@qq.com>
Date: Fri, 2 Jun 2023 01:14:43 +0800
Subject: [PATCH] treble-gms-common

---
 common/common-vendor.mk | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/common/common-vendor.mk b/common/common-vendor.mk
index f4ac093..3ed696d 100644
--- a/common/common-vendor.mk
+++ b/common/common-vendor.mk
@@ -58,8 +58,8 @@ PRODUCT_PACKAGES += \
     AndroidPlatformServices \
     WellbeingPrebuilt \
     GoogleServicesFramework \
-    GoogleTTS \
-    Velvet
+    CalendarGooglePrebuilt \
+    GoogleTTS
 
 ifeq ($(strip $(TARGET_CORE_GMS_EXTRAS)),true)
 PRODUCT_PACKAGES += \
@@ -96,8 +96,6 @@ PRODUCT_PACKAGES += \
     LocationHistoryPrebuilt \
     MarkupGoogle \
     Photos \
-    PrebuiltDeskClockGoogle \
-    AndroidAutoStubPrebuilt \
     ConfigUpdater \
     GoogleOneTimeInitializer \
     GoogleRestorePrebuilt \
@@ -105,11 +103,8 @@ PRODUCT_PACKAGES += \
     Phonesky \
     PrebuiltGmsCoreSc \
     AndroidPlatformServices \
-    SetupWizardPrebuilt \
-    Velvet \
     WellbeingPrebuilt \
-    GoogleServicesFramework \
-    PixelSetupWizard
+    GoogleServicesFramework
 
 endif
 
-- 
2.40.1

